RIPS/rip-7711.md

```diff
@@ -148,11 +148,16 @@ Using `SELFDESTRUCT` is only allowed within the same **frame** the contract was
 UNUSED_GAS_PENALTY = 10
 ```
 
-A penalty of `UNUSED_GAS_PENALTY` percent of the entire unused "execution gas limit" is charged from the
+A penalty of `UNUSED_GAS_PENALTY` percent of the unused `callGasLimit` and `paymasterPostOpGasLimit` is charged from the
 transaction `sender` or `paymaster`.
 
-The "execution gas limit" is calculated as `executionGasLimit = callGasLimit + paymasterPostOpGasLimit`.\
-The unused gas is calculated as `unusedGas = executionGasLimit - gasUsedByExecution - gasUsedByPaymasterPostOp`.
+This penalty is applied to the execution frame and the `postPaymasterTransaction` separately.
+The unused gas is calculated as following for the corresponding frames:
+
+```
+uint256 unusedGasExecution = executionGasLimit - gasUsedByExecution;
```

Comment:
`unusedExecutionGasPenalty` is added to the `actualGasUsed` reported to the `postPaymasterTransaction`

Reply:
Added this elaboration.